src/SlideImage.php

```diff
@@ -63,13 +62,6 @@ class SlideImage extends DataObject implements PermissionProvider
     private static $default_sort = 'SortOrder';
 
     /**
-     * Adds Publish button to SlideImage record
-     *
-     * @var bool
-     */
-    private static $versioned_gridfield_extensions = true;
```

Comment:
let's keep this in for now, was an adjustment for versioning in SilverStripe 4 rc1